src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,123 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+
+  handleTitle = (event) => {
+    this.setState({
+      title: event.target.value,
+    });
+  }
+
+  handleDescription = (event) => {
+    this.setState({
+      description: event.target.value,
+    });
+  }
+
+  handleImgUrl = (event) => {
+    this.setState({
+      imgUrl: event.target.value,
+    });
+  }
+
+  handleImdbUrl = (event) => {
+    this.setState({
+      imdbUrl: event.target.value,
+    });
+  }
+
+  handleImdbId = (event) => {
+    this.setState({
+      imdbId: event.target.value,
+    });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const { title, description, imgUrl, imdbUrl, imdbId } = this.state;
+
+    const movie = {
+      title,
+      description,
+      imgUrl,
+      imdbUrl,
+      imdbId,
+    };
+
+    this.props.addMovie(movie);
+
+    this.setState({
+      title: '',
+      description: '',
+      imgUrl: '',
+      imdbUrl: '',
+      imdbId: '',
+    });
+  }
 
   render() {
+    const { title, description, imgUrl, imdbUrl, imdbId } = this.state;
+
     return (
-      <form>
-        Put the form here
-      </form>
+      <>
+        <h2 className="display-4 text-center mb-3">Add movie</h2>
+        <form
+          onSubmit={this.handleSubmit}
+        >
+          <input
+            className="form-control mb-3"
+            placeholder="Title"
+            required
+            value={title}
+            onChange={this.handleTitle}
```

Comment:
И просто передавать название поля в параметрах
```suggestion
            onChange={(event) => {
              this.handleChange(event, 'title');
            }}
```